docs/getting-started/new-project-guide/python_lang.md

```diff
@@ -27,12 +27,33 @@ Python fuzzing in OSS-Fuzz depends on
 `atheris` package, and dependencies are pre-installed on the OSS-Fuzz base
 docker images.
 
+## Hypothesis
```

Comment:
This makes it slightly confusing to put Hypothesis between Atheris description and its required project file changes for Atheris. I think it is better to move this to end, after "Project files" to its own section. THat way, you give Hypothesis description, required change to go in dockerfile, give a simple example where hypothesis value comes in and then at end give hyperlink to ujson fuzzer with hypothesis. wdyt ? 

## Hypothesis - Property-based testing

Have same description

then say, like to use Hypothesis, add it in Dockerfile

```shell
pip3 install hypothesis
```

maybe show a simple example

And then give hyperlink to this ujson example fuzzer, so people can look at actual code of a complicated one